darshan-util/pydarshan/darshan/experimental/plots/plot_io_cost.py

```diff
@@ -39,11 +39,16 @@ def get_by_avg_series(df: Any, mod_key: str, nprocs: int) -> Any:
         f"{mod_key}_F_WRITE_TIME",
         f"{mod_key}_F_META_TIME",
     ]
+    if "PNETCDF_FILE" in mod_key:
+        cols.append("PNETCDF_FILE_F_WAIT_TIME")
+    else:
+        cols.append("Wait")
```

Comment:
Is the else statement here just needed to ensure `cols` has the right size or something? Just double checking that's as expected here, since the column name is not actually a counter.

Reply:
It just makes a column of all-zero values yeah. What worried me a bit, as suggested above, is that the test file had zeros there anyway, so programmer paranoia about testing the genuine code path.

Reply:
Sounds good, just wanted to check.